pkg/cmd/cli/cmd/newapp.go

```diff
@@ -90,6 +90,19 @@ application is created.`
 
   // Search for "ruby" in stored templates and print the output as an YAML
   $ %[1]s new-app --search --template=ruby --output=yaml`
+
+	newAppNoInput = `You must specify one or more images, image streams, templates or source code locations to create an application.
+
+Suggestions:
+Use '-L' or '--list' flag to list all local templates and image streams that can be used to create an application
```

Comment:
Can we take out 15 words of the latter so that it takes up 3 less lines?

On Wed, Aug 26, 2015 at 12:31 PM, Fabiano Franz <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/newapp.go
> <https://github.com/openshift/origin/pull/4375#discussion_r37996372>:
>
> > @@ -90,6 +90,19 @@ application is created.`
> >
> >    // Search for "ruby" in stored templates and print the output as an YAML
> >    $ %[1]s new-app --search --template=ruby --output=yaml`
> > +
> > +	newAppNoInput = `You must specify one or more images, image streams, templates or source code locations to create an application.
> > +
> > +Suggestions:
> > +Use '-L' or '--list' flag to list all local templates and image streams that can be used to create an application
>
> I agree this needs to be compressed a little bit. I'd say something like
>
> You must specify one or more images, image streams, templates or source code locations to create an application.
>
> To list all local templates and image streams that can be used to create an application, use 'oc new-app -L'.
>
> You can also use '-S' to search all templates, image streams, and Docker images that match the arguments provided, for instance 'oc new-app -S php' or 'oc new-app -S --template=ruby'.
>
> or
>
> You must specify one or more images, image streams, templates or source code locations to create an application.
>
> To list all local templates and image streams that can be used to create an application, use
>
>   $ oc new-app -L
>
> You can also use '-S' to search all templates, image streams, and Docker images that match the arguments provided.
>
>   $ oc new-app -S php
>   $ oc new-app -S --template=ruby
>   $ oc new-app -S --image=mysql
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4375/files#r37996372>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift